src/util/config.js

```diff
@@ -18,7 +18,7 @@ function loadConfig() {
 
     // package major will be 0 until release
     if (sv.major == 0) {
-      version = `beta-1.0b${sv.patch}`;
+      version = `beta-0.1.${sv.patch}`;
```

Comment:
This is an aberration I caused to get the Windows Installer build correctly. However we're probably moving to rc1 next.

Reply:
I figure I'll keep that little bit of code synced to the latest version until release. If for no other reason than I need it and if anyone else uses this fork they'll need it too